src/js/entities-service/entities/actions.js

```diff
@@ -39,6 +39,9 @@ const _Model = BaseModel.extend({
     DetailsChanged({ details, attributes = {} }) {
       this.set({ details, ...attributes });
     },
+    SharingUpdated({ sharing, outreach, attributes = {} }) {
```

Comment:
so a bunch of these are wrong.. the only thing doing any work here is attributes

Reply:
These have been updated 👍